packages/bootstrap4-theme/src/scss/extends/_image-based-cards.scss

```diff
@@ -0,0 +1,146 @@
+.image-based-card {
```

Comment:
When it's not a class name inherited by Bootstrap we have a convention of prefixing class names for Unity components with `uds-` so we avoid accidental collisions with other CMS/application classes and unexpected BS class names.  Can you change this to `.uds-image-based-card` and check for other class names that should be prefixed?